CDP4JsonFileDal.Tests/JsonFileDalTestFixture.cs

```diff
@@ -99,12 +109,23 @@ public void OneTimeSetUp()
         public void SetUp()
         {
             var path = Path.Combine(TestContext.CurrentContext.TestDirectory, "files", "LOFT_ECSS-E-TM-10-25_AnnexC.zip");
+            var migrationSourceFile = Path.Combine(TestContext.CurrentContext.TestDirectory, @"..\..\..\files", "migration.json");

```

Comment:
I might not be understanding this correctly, but does this file path point outside of the project?

Reply:
TestContext.CurrentContext.TestDirectory point to test project source path /bin/$(SolutionConfiguration)/${TargetFramework} and files path its local path inside test project.